Update HIP definition (#2134)
ROCm/rocm-systems: projects/hip/docs/how-to/hip_runtime_api.rst

********************************************************************************

> The HIP runtime API provides C and C++ functionalities to manage event, stream,

Reviewer: ```suggestion
The HIP runtime API provides C and C++ functionality to manage events, streams,
```

Reviewer: Might be wrong here with this edit

Reviewer 2: I would leave it as is.

and memory on GPUs. The HIP runtime uses :doc:`Compute Language Runtime (CLR) <../understand/amd_clr>`.

> **CLR** contains source code for AMD's compute language runtimes: ``HIP`` and

Reviewer: ```suggestion
CLR contains source code for AMD's compute language runtimes: ``HIP`` and
```

Reviewer: Is it AMD or ROCm?
this sentence sounds like all of AMD

Reviewer 2: Maybe ROCm is better.

Reviewer 3: Added AMD ROCm

``OpenCL™``. CLR includes the ``HIP`` implementation on the AMD
platform: `hipamd <https://github.com/ROCm/rocm-systems/tree/develop/projects/clr/hipamd>`_ and the
ROCm Compute Language Runtime (``rocclr``). ``rocclr`` is a
virtual device interface that enables the HIP runtime to interact with
different backends such as :doc:`ROCr <rocr-runtime:index>` on Linux or PAL on
Windows. CLR also includes the `OpenCL runtime <https://github.com/ROCm/rocm-systems/tree/develop/projects/clr/opencl>`_

> implementation.

Reviewer: ```suggestion
``OpenCL™``. CLR includes the ``HIP`` implementation on the AMD
platform: `hipamd <https://github.com/ROCm/rocm-systems/tree/develop/projects/clr/hipamd>`_ and the
ROCm Compute Language Runtime (``rocclr``). ``rocclr`` is a
virtual device interface that enables the HIP runtime to interact with
different backends, such as :doc:`ROCr <rocr-runtime:index>` on Linux or PAL on
Microsoft Windows. CLR also includes the `OpenCL runtime <https://github.com/ROCm/rocm-systems/tree/develop/projects/clr/opencl>`_
implementation.
```

Reviewer: Is it AMD platform or ROCm platform?

Reviewer 2: I think ROCm platform

Reviewer 3: Added AMD ROCm platform

> The backends of HIP runtime API are summarized in the following figure:

Reviewer: ```suggestion
The HIP runtime API backends are summarized in the following figure:
```

Reviewer 3: Fixed
